websites/management/commands/backfill_video_file_size.py

```diff
@@ -0,0 +1,66 @@
+"""Backfill video file size using archive url"""  # noqa: INP001
+
+from main.management.commands.filter import WebsiteFilterCommand
+from videos.tasks import populate_video_file_size
+from websites.models import Website, WebsiteContent
+
+CHUNK_SIZE = 50
+
+
+class Command(WebsiteFilterCommand):
+    """Backfill metadata['file_size'] for video resources using archive_url"""
+
+    help = __doc__
+
+    def add_arguments(self, parser):
+        super().add_arguments(parser)
+        parser.add_argument(
+            "--sync",
+            action="store_true",
+            help="Run the backfill synchronously instead of queuing tasks.",
+        )
+
+    def handle(self, *args, **options):
+        super().handle(*args, **options)
+
+        sync_execution = options.get("sync", False)
+
+        websites = Website.objects.all()
+        websites = self.filter_websites(websites)
+
+        videos = WebsiteContent.objects.filter(
+            type="resource", metadata__resourcetype="Video", website__in=websites
```

Comment:
nitpick - use constant from `websites.constants`
```suggestion
            type=CONTENT_TYPE_RESOURCE, metadata__resourcetype=RESOURCE_TYPE_VIDEO, website__in=websites
```